src/utils/popoverMixins.js

```diff
@@ -158,6 +166,17 @@ export default {
     adjustArrow (delta, dimension, isVertical) {
       this.$refs.arrow.style[isVertical ? 'left' : 'top'] = 50 * (1 - delta / dimension) + '%'
       this.$refs.arrow.style[isVertical ? 'top' : 'left'] = ''
+      let translateLeft = 0;
+      let translateTop = 0;
+      if (this.placement === 'left' || this.placement === 'right') {
+        translateTop = this.isPopover ? -75 : -50;
+      }
+
```

Comment:
This javascript file does not leave blank lines inside methods. Let's remove it to stay consistent in the coding style.